animalfacts.py

```diff
@@ -1322,6 +1322,17 @@ def animalfactsbot(reddit):
     'Ladybugs got their name from Catholic farmers, who prayed to the Virgin Mary to save their crops from pests. When appeared and ate the pests they were called “the beetles of our lady”.',
     )
 
+LAMPREY_FACTS = (
+    'The lamprey is a jawless fish with a terrifying mouth found in both saltwater and freshwater systems in most temperate regions of our planet.'
+    'The sea lamprey is considered to be a parasitic fish because it bores holes into its prey while feeding on their blood and bodily fluids, slowly killing their prey and damaging the species overall survival rate'
+    'a single parasitic lamprey can kill over 40 pounds or more of fish in its lifetime'
+    'lampreys have sometimes inaccurately been called "lamprey eels," but they are actually more closely related to sharks'
+    'lamprey belongs to a superclass of jawless fish known agnathans. The unique thing about the agnathan class is that it’s a prehistoric group that is largely extinct'
+    'The proteins in their saliva widen the blood vessels of their prey while their abrasive tongue and piercing teeth damage the skin of its victim and induce blood flow'
+    'Lamprey breeds in high quality, deep, fast flowing rivers with clean gravel in which to spawn and a sandy substrate for the larvae to burrow into'
+    'Lampreys possess vertebral structures called arcualia, tiny cartilaginous skeletal elements that are homologous with the neural arches of vertebrates.'
+    )
```

Comment:
'Spawning takes place in the spring. Sea lamprey can deposit as much as 100 000 eggs in the stony nests. Sea lampreys will die immediately after spawning.',